Tidify/Targets/Tidify/Sources/AppDelegate.swift

```diff
@@ -50,6 +51,13 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 }
 
 private extension AppDelegate {
+  func setupNavigationBar() {
+    UIBarButtonItem.appearance().setTitleTextAttributes([.foregroundColor: UIColor.clear], for: .normal)
+    UINavigationBar.appearance().titleTextAttributes = [.foregroundColor: UIColor.t_gray(weight: 800)]
+    UINavigationBar.appearance().backIndicatorImage = UIImage(named: "backButtonIcon")
+    UINavigationBar.appearance().backIndicatorTransitionMaskImage = UIImage(named: "backButtonIcon")
+  }
+
```

Comment:
```suggestion
    let navigationBar: UINavigationBar = UINavigationBar.appearance()
    UIBarButtonItem.appearance().setTitleTextAttributes([.foregroundColor: UIColor.clear], for: .normal)
    navigationBar.titleTextAttributes = [.foregroundColor: UIColor.t_gray(weight: 800)]
    navigationBar.backIndicatorImage = UIImage(named: "backButtonIcon")
    navigationBar.backIndicatorTransitionMaskImage = UIImage(named: "backButtonIcon")
```

`UINavigationBar`의 `appearance` 객체를 매번 생성할 필요가 없다면 위처럼 해도 좋을듯!

Reply:
오 이건 생각 못해봤네 ㅋㅋㅋ 땡큐!!